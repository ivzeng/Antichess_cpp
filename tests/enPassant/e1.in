d2d4
b7b5
d4d5
b5b4
a2a4
b4a3
b1a3
undo
undo
undo
c2c4
b4c3
b1c3
undo
undo
undo