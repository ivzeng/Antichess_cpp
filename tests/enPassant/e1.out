round0:
8 R N B Q K B N R 
7 P P P P P P P P 
6 - - - - - - - - 
5 - - - - - - - - 
4 - - - - - - - - 
3 - - - - - - - - 
2 p p p p p p p p 
1 r n b q k b n r 
  a b c d e f g h 
player(1)'s turn:
capture moves: 
placeful moves: b1c3 b1a3 g1h3 g1f3 a2a4 a2a3 b2b4 b2b3 c2c4 c2c3 d2d4 d2d3 e2e4 e2e3 f2f4 f2f3 g2g4 g2g3 h2h4 h2h3 
Basic::act() is called
round0:
8 R N B Q K B N R 
7 P P P P P P P P 
6 - - - - - - - - 
5 - - - - - - - - 
4 - - - p - - - - 
3 - - - - - - - - 
2 p p p - p p p p 
1 r n b q k b n r 
  a b c d e f g h 
player(2)'s turn:
capture moves: 
placeful moves: b8c6 b8a6 g8h6 g8f6 a7a5 a7a6 b7b5 b7b6 c7c5 c7c6 d7d5 d7d6 e7e5 e7e6 f7f5 f7f6 g7g5 g7g6 h7h5 h7h6 
Basic::act() is called
round1:
8 R N B Q K B N R 
7 P - P P P P P P 
6 - - - - - - - - 
5 - P - - - - - - 
4 - - - p - - - - 
3 - - - - - - - - 
2 p p p - p p p p 
1 r n b q k b n r 
  a b c d e f g h 
player(1)'s turn:
capture moves: 
placeful moves: e1d2 d1d2 d1d3 c1d2 c1e3 c1f4 c1g5 c1h6 b1d2 b1c3 b1a3 g1h3 g1f3 a2a4 a2a3 b2b4 b2b3 c2c4 c2c3 d4d5 e2e4 e2e3 f2f4 f2f3 g2g4 g2g3 h2h4 h2h3 
Basic::act() is called
round1:
8 R N B Q K B N R 
7 P - P P P P P P 
6 - - - - - - - - 
5 - P - p - - - - 
4 - - - - - - - - 
3 - - - - - - - - 
2 p p p - p p p p 
1 r n b q k b n r 
  a b c d e f g h 
player(2)'s turn:
capture moves: 
placeful moves: c8b7 c8a6 b8c6 b8a6 g8h6 g8f6 a7a5 a7a6 b5b4 c7c5 c7c6 d7d6 e7e5 e7e6 f7f5 f7f6 g7g5 g7g6 h7h5 h7h6 
Basic::act() is called
round2:
8 R N B Q K B N R 
7 P - P P P P P P 
6 - - - - - - - - 
5 - - - p - - - - 
4 - P - - - - - - 
3 - - - - - - - - 
2 p p p - p p p p 
1 r n b q k b n r 
  a b c d e f g h 
player(1)'s turn:
capture moves: 
placeful moves: e1d2 d1d2 d1d3 d1d4 c1d2 c1e3 c1f4 c1g5 c1h6 b1d2 b1c3 b1a3 g1h3 g1f3 a2a4 a2a3 b2b3 c2c4 c2c3 d5d6 e2e4 e2e3 f2f4 f2f3 g2g4 g2g3 h2h4 h2h3 
